src/js/pagination.js

```diff
@@ -0,0 +1,70 @@
+import { showError, showButton, hideButton, setActualPage } from './dom.js';
+import { checkError } from './utils.js';
+
+export function initializePagination(numberOfPages) {
+    document.getElementById('total-pages').textContent = numberOfPages;
+    document.getElementById('previous-button').classList.add('hidden');
+    setActualPage(1);
+}
+
+export function handlePagination(event, callbacks) {
+    event.preventDefault();
+    const actualPage = Number(document.getElementById('pagination').dataset.selected);
+    const totalPages = Number(document.getElementById('total-pages').textContent);
+    const buttonClicked = event.target.dataset.button;
+    // const pageInfo = {actualPage, totalPages }
+    if (buttonClicked === "next") {
+        goNextPage(actualPage, totalPages, callbacks);
+
+    } else if (buttonClicked === "previous") {
+        goPreviousPage(actualPage, totalPages, callbacks);
+
+    } else if (buttonClicked === "seek"){
+        const $selection = document.getElementById('page-selection')
+        const destiny = Number($selection.value);
```

Comment:
This could be `const $selection = Number(document.getElementById('page-selection').value)` and if you want to pass showError a string you can call `.toString` method.

Reply:
showError() receives a HTML element. But maybe that's a mistake since i'm using it only here.